src/mmw/apps/home/urls.py

```diff
@@ -4,7 +4,7 @@
 from __future__ import division
 
 from django.conf.urls import patterns, url
-from apps.home.views import home_page, project, compare
+from apps.home.views import home_page, project, compare, base_layers
```

Comment:
As per @mmcfarland I'd strongly favor having this come in with the initial page rather than as a separate request.